docs/code_ref/coordinates.rst

```diff
@@ -4,13 +4,6 @@ SunPy coordinates
 The SunPy coordinates submodule is an implementation of the common solar physics
 coordinate frames using the :ref:`Astropy coordinates framework <astropy:astropy-coordinates>`.
 
-.. note::
-
-   The accuracy of the transformations in this module have not been rigorously
```

Comment:
I wouldn't claim that I've tested the transformations *rigorously*, but I think the disclaimer as is makes the code sound a lot less tested than it actually is, so I support removal.